src/web/transactions.rs

```diff
@@ -144,14 +119,17 @@ impl From<Executable> for ExecutableDTO {
 }
 
 #[get("/{hash}")]
+#[allow(deprecated)]
```

Comment:
This works, but you apply this attribute to the whole function body. You might be more precise and but the attribute right next to the piece of code which causes the warning.